NEWS.md

```diff
@@ -1,6 +1,10 @@
 
 # tidyselect 0.2.5.9000
 
+* `vars_select()` now supports duplicate variables (#94). They are
+  ignored if not part of the selection. Otherwise, they are repaired
+  to make them unique.
+
```

Comment:
The thing that feels odd is I perceive `vars_select()` as being "read only", i.e. changing the names that it's operating on is out of scope.

Reply:
You can change the names with `data %>% dplyr::select(foo = bar)`. The main difference between `select()` and `rename()` is that the former is a shrinking verb while the latter maintains the shape.